CMakeLists.txt

```diff
@@ -184,6 +186,25 @@ foreach(backend ${built_backends})
     target_compile_definitions(${backend}
       PRIVATE AF_WITH_LOGGING)
   endif()
+
+  if(AF_WITH_STACKTRACE)
+    if(UNIX)
+      find_library(Backtrace_LIBRARY backtrace
+        DOC "libbacktrace.so file for more informative stacktraces. https://github.com/ianlancetaylor/libbacktrace")
+      if(Backtrace_LIBRARY)
+        target_link_libraries(${backend}
+          PRIVATE ${Backtrace_LIBRARY})
+        target_compile_definitions(${backend}
+          PRIVATE
+            BOOST_STACKTRACE_USE_BACKTRACE)
+      endif()
+    endif()
+  else()
```

Comment:
Does this mean we can't use backtrace feature on Windows ?

Reply:
Its enabled on windows. It doesn't need special flags to be enabled.